cbmgmtx/errors.go

```diff
@@ -59,18 +59,18 @@ func (e ServerInvalidArgError) Error() string {
 	return fmt.Sprintf("%s: %s - %s", e.Unwrap().Error(), e.Argument, e.Reason)
 }
 
-type ResourceError struct {
+type ErrorWithResourceDetail struct {
```

Comment:
This feels a bit like it's not in keeping with our other error types, although I don't have a suggestion for a better name...

Reply:
Thoughts on the new name?